cortex-mixin/alerts/alerts.libsonnet

```
@@ -368,7 +368,7 @@
           },
           annotations: {
             message: |||
-              Too much memory being used by ingesters - add more ingesters.
+              Too much memory being used by {{ $labels.instance }}, - add more ingesters.
```

`, -` looks too much 😉 

```suggestion
              Too much memory being used by {{ $labels.instance }} - add more ingesters.
```

---

Oops, a typo (I should stop hurrying :))